wallet/wallet-controller/src/lib.rs

```diff
@@ -296,7 +311,7 @@ impl<T: NodeInterface + Clone + Send + Sync + 'static> Controller<T> {
             }
 
             if self.staking_started {
-                let generate_res = self.generate_block(None).await;
+                let generate_res = self.generate_block(DEFAULT_ACCOUNT_INDEX, None).await;
```

Comment:
Is staking on the `DEFAULT` account always the case? Should this be a parameter instead?